src/components/clusterprovider/azure/azureclusterprovider.ts

```diff
@@ -353,6 +356,8 @@ function renderConfigurationResult(configureResult: ActionResult<azure.Configure
     const configResult = result.result;
     const clusterServiceString = result.result.clusterType === "aks" ? "Azure Kubernetes Service" : "Azure Container Service";
 
+    reporter.sendTelemetryEvent("addCluster", { result: configureResult.result.succeeded ? "success" : "fail", clusterType: clusterServiceString });
```

Comment:
`clusterServiceString` is the long name ('Azure Kubernetes Service'); elsewhere telemetry uses the cluster type ID (e.g. `aks`).